core/src/apps/zcash/orchard/crypto/address.py

```diff
@@ -0,0 +1,41 @@
+from typing import TYPE_CHECKING
+
+from trezor.crypto.pallas import Point, group_hash
+
+if TYPE_CHECKING:
+    from trezor.crypto.pallas import Scalar
+
+    pass  # address.i
```

Comment:
this was solving some mPy bug

```py
if TYPE_CHECKING:
    from trezor.crypto.pallas import Scalar

...
```

for TYPE_CHECKING=False is translated into `address.i` as
```py
if TYPE_CHECKING:
    # from trezor.crypto.pallas import Scalar

...
```

which causes syntax error

let me see whether this bug will emerge again

Reply:
18aa95013d2de0f6cda2bf6886d43d77c9ec28e4

Reply:
it would only happen if that import was `typing`, i.e.:
```
if TYPE_CHECKING:
    # from typing import Any
```
non-typing imports will remain there

Reply:
ack